tests/unit/via/views_test.py

```diff
@@ -57,37 +49,6 @@ class TestIncludeMe:
 
 
 class TestPdfRoute:
-    @pytest.mark.parametrize(
-        "template_content",
-        [
-            'window.PDF_URL = "https://via3.hypothes.is/proxy/static/http://example.com";',
-            'window.CLIENT_EMBED_URL = "https://hypothes.is/embed.js";',
-            "openSidebar: true",
-            "requestConfigFromFrame: 'https://lms.hypothes.is'",
-        ],
-        ids=[
-            "sets PDF_URL to proxied pdf url",
-            "sets CLIENT_EMBED_URL to client embed url",
-            "configures open_sidebar client setting",
-            "configures open_sidebar client setting",
-        ],
-    )
-    def test_pdf_renders_parameters_in_pdf_template(
-        self, template_content, template_env
-    ):
-        template = template_env.get_template("pdf_viewer.html.jinja2")
-        html = template.render(
-            {
-                "pdf_url": "https://via3.hypothes.is/proxy/static/http://example.com",
-                "client_embed_url": "https://hypothes.is/embed.js",
-                "h_open_sidebar": True,
-                "h_request_config": "https://lms.hypothes.is",
-            }
-        )
-        tree = BeautifulSoup(html, features="html.parser")
-
-        assert template_content in str(tree.head)
-
```

Comment:
Not only does this break when we change the template, I just don't think we should be doing it. If we care about something like this we should probably add a functional test.

Reply:
:+1: We should just write functional tests